e2e/tests/ui-driven/src/mocks/gisResponse.ts

```diff
@@ -17,7 +18,7 @@ export async function setupGISMockResponse(page: Page) {
 export function checkGISMockRequestUrl(url: string) {
   const splitUrl = url.split("/").pop()?.split("%2C");
   return (
-    !splitUrl?.includes("designated.conservationArea") &&
+    splitUrl?.includes("designated.conservationArea") &&
```

Comment:
Again, this change was required in order to get tests to pass but it seems like a fairly basic change (`true` → `false`).

`setupGISMockResponse()` is only used in a single context, and it must resolve to `true`.

Any ideas what's caused this change?

Reply:
Nothing immediate comes to mind here ! On the editor side, the component is setup to check this/all datasets, right ?

Reply:
Yep as far as I can tell - it makes a lot more sense that both these conditions would be `true`.